src/plugins/vis_type_vega/public/data_model/vega_parser.ts

```diff
@@ -137,6 +138,8 @@ The URL is an identifier only. Kibana and your browser will never access this UR
 
     this._config = this._parseConfig();
     this.hideWarnings = !!this._config.hideWarnings;
+    this._parseBool('restoreSignalValuesOnRefresh', this._config, false);
+    this.restoreSignalValuesOnRefresh = this._config.restoreSignalValuesOnRefresh;
```

Comment:
This config works well, and I tested it with my simple clicker example and the Vega tutorial. The Vega tutorial works correctly by default, and does not work when I turn on signal refreshing.